.gitignore

```diff
@@ -0,0 +1 @@
+*.code-workspace
```

Comment:
Do we want to keep this or perhaps provide a more comprehensive git ignore file on a separate PR?